bsc#1182983 gpg: out of core handler ignored in FIPS mode while typing Tab key to Auto-Completion

Index: libgcrypt-1.10.0/src/global.c
===================================================================
--- libgcrypt-1.10.0.orig/src/global.c
+++ libgcrypt-1.10.0/src/global.c
@@ -974,7 +974,6 @@ _gcry_set_outofcore_handler (int (*f)(vo
 
   if (fips_mode () )
     {
-      log_info ("out of core handler ignored in FIPS mode\n");
       return;
     }
 
